packages/cli/src/cmds/init/initUtils.test.ts

```diff
@@ -72,16 +71,15 @@ describe('initUtils', () => {
       expect(global.console.log).toHaveBeenCalledTimes(2);
     });
 
-    it('throws error if unable to parse packagejson', async () => {
+    it('throws error if unable to parse package.json', async () => {
       const existsSyncMock = jest
         .spyOn(fs, 'existsSync')
         .mockImplementation(() => true);
-      const readFileMock = jest
```

Comment:
This changes the test, is that intentional?

Reply:
What do you mean?

Reply:
We now test that `readJsonFile` fails instead of `JSON.parse` which is different. It may be fine, I just wanted to point it out regardless

Reply:
It just tests the error handling behaviour, so it doesn't really matter where it throws. I think I changed it because `JSON.parse` wasn't called anymore in the first place.